examples/src/main/java/org/bitcoinj/examples/ForwardingService.java

```diff
@@ -72,7 +76,24 @@ public static void main(String[] args) throws Exception {
         forwardingService.start();
 
         // Start listening and forwarding
-        forwardingService.forward();
+        CompletableFuture<Transaction> forwardedTxFuture = forwardingService.forward()
+                .whenComplete(ForwardingService::logForwardingBroadcastComplete);
+
+        System.out.println("Wallet will receive coins on:" + forwardingService.receivingAddress());
+        System.out.println("Will send coins to: " + address);
+        System.out.println("Waiting for coins to arrive. Press Ctrl-C to quit.");
+
+        // Wait for the forwarding transaction to be broadcast
+        forwardedTxFuture.wait(Duration.ofDays(7).toMillis());
```

Comment:
This is a mistake (`.wait` is the method on `Object`!). We should be calling `forwardedTxFuture.get()` with a timeout.